### PR TITLE
Code formatting and fix yuidoc #3

### PR DESCRIPTION
**Pull request by SeregaMatin**
[TFS 108076]

### CODE REVIEW

File: addon/components/colsconfig-dialog-content.js [hunk 1]
```diff
@@ -9,11 +9,10 @@ import FlexberryBaseComponent from './flexberry-base-component';
  * @extends FlexberryBaseComponent
```
Comment (hvostya) [inline addon/components/colsconfig-dialog-content.js]: Should there be added comments on this file too?

File: addon/components/flexberry-file.js [hunk 2]
```diff
@@ -9,64 +9,34 @@ import { translationMacro as t } from 'ember-i18n';
 /**
   Flexberry file component.
 
+  Usage sample:
```
Comment (hvostya) [inline addon/components/flexberry-file.js]: You can use @example.

File: addon/components/flexberry-file.js [hunk 3]
```diff
@@ -9,64 +9,34 @@ import { translationMacro as t } from 'ember-i18n';
...
+
   @class FlexberryFileComponent
   @extends FlexberryBaseComponent
 */
 export default FlexberryBaseComponent.extend({
   /**
     Selected file content. It can be used as source for image tag in order to view preview.
 
-    @property _previewImageAsBase64String
     @private
```
Comment (hvostya) [inline addon/components/flexberry-file.js]: @yzaripova says that @private should be after @property.

File: addon/components/flexberry-file.js [hunk 4]
```diff
@@ -139,249 +104,302 @@ export default FlexberryBaseComponent.extend({
...
+  showPreview: false,
 
-    return !(uploadIsInProgress || Ember.isNone(jsonInitialValue));
-  }),
+  /**
+    Flag: indicates whether to show upload button or not.
+
+    @property showUploadButton
+    @type Boolean
+    @default false
```
Comment (hvostya) [inline addon/components/flexberry-file.js]: Is default value correct?

File: addon/components/flexberry-file.js [hunk 5]
```diff
@@ -139,249 +104,302 @@ export default FlexberryBaseComponent.extend({
...
+  */
+  showUploadButton: undefined,
+
+  /**
+    Flag: indicates whether to show download button or not.
+
+    @property showDownloadButton
+    @type Boolean
+    @default true
```
Comment (hvostya) [inline addon/components/flexberry-file.js]: Is default value correct?

File: addon/mixins/olv-toolbar-mixin.js [hunk 6]
```diff
@@ -2,7 +2,7 @@ import Ember from 'ember';
 
 // TODO: rename file, add 'controller' word into filename.
 export default Ember.Mixin.create({
-  _userSettingsService: Ember.inject.service('user-settings-service'),
+  _userSettingsService: Ember.inject.service('user-settings'),
```
Comment (hvostya) [inline addon/mixins/olv-toolbar-mixin.js]: Comments?

File: addon/resolver.js [hunk 7]
```diff
@@ -1,67 +1,82 @@
...
   /**
-   * Checks if template or classs with given full name is known and could be resolved.
-   */
-  isKnown: function(fullName) {
-    var type = fullName.split(':')[0];
-    var knownForType = this.knownForType(type) || {};
+    Checks if template or class with given full name is known and could be resolved.
+
+    @method isKnown
+    @param {String} fillName Resource full name (with path inside type-related directory).
```
Comment (hvostya) [inline addon/resolver.js]: fullName

File: addon/components/mobile/flexberry-file.js [hunk 8]
```diff
@@ -12,17 +12,69 @@ import FlexberryFile from './../flexberry-file';
   @extends FlexberryFileComponent
```
Comment (hvostya) [inline addon/components/mobile/flexberry-file.js]: /**
   @module ember-flexberry
  */ <= extra space

File: addon/components/colsconfig-dialog-content.js (hunk 1 above)
Comment (SeregaMatin) [inline addon/components/colsconfig-dialog-content.js]: They should, but this issue is not in responsibility area of [TFS 108076] task.

File: addon/components/flexberry-file.js (hunk 2 above)
Comment (SeregaMatin) [inline addon/components/flexberry-file.js]: Then example will not appear here:
image

File: addon/components/flexberry-file.js (hunk 3 above)
Comment (SeregaMatin) [inline addon/components/flexberry-file.js]: It won't cause any effects on generated docs, but ok, l'll fix it.

File: addon/components/flexberry-file.js (hunk 4 above)
Comment (SeregaMatin) [inline addon/components/flexberry-file.js]: Yes: https://github.com/Flexberry/ember-flexberry/blob/code-formatting-and-fix-yuidoc-%233/addon/components/flexberry-file.js#L518
And we had already discussed it in one of the previous pull-requests.

File: addon/mixins/olv-toolbar-mixin.js (hunk 6 above)
Comment (SeregaMatin) [inline addon/mixins/olv-toolbar-mixin.js]: Comments for this class is not in responsibility area of [TFS 108076] task.

File: addon/components/flexberry-file.js (hunk 5 above)
Comment (SeregaMatin) [inline addon/components/flexberry-file.js]: Yes: https://github.com/Flexberry/ember-flexberry/blob/code-formatting-and-fix-yuidoc-%233/addon/components/flexberry-file.js#L519
And we had already discussed it in one of the previous pull-requests.

File: addon/resolver.js (hunk 7 above)
Comment (SeregaMatin) [inline addon/resolver.js]: Right, i'll fix it.

File: addon/components/mobile/flexberry-file.js (hunk 8 above)
Comment (SeregaMatin) [inline addon/components/mobile/flexberry-file.js]: Ok, i'll fix it.

File: addon/components/flexberry-file.js (hunk 4 above)
Comment (hvostya) [inline addon/components/flexberry-file.js]: But new info on comment didn't appear so I will ask it each time.

File: addon/components/flexberry-file.js (hunk 5 above)
Comment (hvostya) [inline addon/components/flexberry-file.js]: But new info on comment didn't appear so I will ask it each time.

File: addon/services/log.js [hunk 9]
```diff
@@ -0,0 +1,348 @@
+/**
```
Comment (hvostya) [inline addon/services/log.js]: Won't there be problems that it is not renamed file but new one?

Comment (SeregaMatin) [inline addon/services/log.js]: Which kind of problems do you mean?

Comment (SeregaMatin) [inline addon/services/log.js]: Actually file was just renamed, but my git client handles it as: old file - remove/new file add.

Comment (hvostya) [inline addon/services/log.js]: Problems on next merges.
For example now there is a branch where "addon/services/flexberry-logging.js" is fixed. If I merge this pull request won't there be merge problems with branch with fixed "addon/services/flexberry-logging.js"?

Comment (SeregaMatin) [inline addon/services/log.js]: Such changes always will cause merge problems.

Comment (ehaberev) [inline addon/services/log.js]: I think we need to ask all developers, whos branches are using user-settings service or log service, to merge `develop` branch into their branches after merge this pull request.

**Pull request merged**